Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/clientcmd/loader.go

```diff
@@ -238,7 +237,6 @@ func LoadFromFile(filename string) (*clientcmdapi.Config, error) {
 	if err != nil {
 		return nil, err
 	}
-	glog.V(3).Infoln("Config loaded from file", filename)
```

Comment:
Why the hate for tracing?  Its set to a pretty high level and useful message, especially when people are setting the env var.